modules/iap-brand/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "application_title" {
```

Comment:
We need a README fie for this module, describing its purpose and giving a simple usage example.